runtime/makelib/targets.mk.aix.inc.ftl

```diff
@@ -92,15 +92,31 @@ ifdef I5_VERSION
 else
   CFLAGS += -s $(UMA_CC_MODE)
 endif
-CFLAGS += -q mbcs -qlanglvl=extended -qarch=ppc -qinfo=pro -qalias=noansi -qxflag=LTOL:LTOL0 -qsuppress=1506-1108
+
+ifeq ($(notdir $(CC)),xlclang)
```

Comment:
This pattern assumes that `CC` is not something like `xlclang-16`.

Reply:
The statement was updated, @keithc-ca any further concerns?

Reply:
Given it appears that xlclang alters its behavior based on the name used to invoke the compiler, this seems reasonable (xlc and xlclang use the same binary).